sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -130,6 +130,137 @@ To update the value of any of the the available settings, we will call the Updat
 Setting updatedSetting
    = settingsClient.UpdateSetting(settingsList.Value[0].Name, options).Value;
 ```
+## Creating a BackupRestoreClient
+
+To create a new `BackupRestoreClient` to perform these operations, you need the endpoint to an Azure Key Vault HSM and credentials.
+
+Key Vault BackupRestoreClient client for C++ currently supports any `TokenCredential` for authenticating.
+
+```cpp Snippet:SampleBackupRestoreCreateCredential
```

Comment:
Are these actually C++ snippets, or is this taken from a different languages snippet logic?

Reply:
i took it from the other samples in keyvault, i made a copy of an existing readme and made updates accordingly, keeping the structure, i don't know exactly what purpose those things serve